src/storage/kv/cursor.rs

```diff
@@ -551,21 +589,71 @@ mod tests {
         ];
 
         for &(ref k, ref v) in &base_data {
-            engines.kv.put(&data_key(k), v).unwrap();
+            engine.put(&data_key(k), v).unwrap();
         }
         (r, base_data)
     }
 
+    #[test]
+    fn test_seek_and_prev_with_prefix_seek() {
```

Comment:
I can't get the purpose of this test...

Reply:
if `min_key` or `max_key` is set when prefix-seek enabled, this test will fail.